dealers_choice.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
Try to organize this html file a little more just by using proper indentation for each html tag.